src/components/OrgHeader.tsx

```diff
@@ -30,6 +53,7 @@ const OrgHeader: FC<{ club: Club }> = ({ club }) => {
         <button className="m-5 rounded-full bg-slate-100 px-4 py-2 font-semibold text-slate-900 transition-colors hover:bg-slate-300">
           Join
         </button>
+        <ContactButtons contact={ dummyData || []}/>
```

Comment:
now that it's updated try passing in the contacts based on the club object since now it is guaranteed to default to an empty array

Reply:
Update: Passed club.contract and successfully loaded one button that connects to "google.com." Also removed dummy data set.

Reply:
I think the google.com is just dummy data I added to the db.